clientbuffer.cpp

```diff
@@ -134,14 +137,16 @@ class CClientBufferMod : public CModule
 void CClientBufferMod::OnAddClientCommand(const CString& line)
 {
     const CString identifier = line.Token(1);
+
     if (identifier.empty()) {
-        PutModule("Usage: AddClient <identifier>");
+        PutModule("Usage: AddClient <identifier> <timelimit>");
```

Comment:
Usage string still has the timelimit parameter:

```suggestion
        PutModule("Usage: AddClient <identifier>");
```